tce/src/applibs/LLVMBackend/TDGen.cc

```diff
@@ -343,6 +343,8 @@ TDGen::analyzeRegisterFileClasses() {
                 if (lane < 8 && lane > highestLaneBool_) {
                     highestLaneBool_ = lane;
                 }
+                //suppress fallthrough warning
```

Comment:
You can remove this comment in case the next line is a magic comment recognized by Clang?